### ISSUE TITLE
Modeler 5.5.0 Execution Listeners: Error when input is cleared (Uncaught in Promise)

### ISSUE THREAD

**Issue by Adrian-WDW** [camunda/camunda-modeler#3303]
### Describe the bug

In a task, when you add an execution listener and set the type to either 'Expression', 'JavaClass' or 'DelegateExpression' and type something in the inputfield beneath, then clear the input with backspace, you get the following error and can't really use the modeler afterwards, so you have to reload/restart.

> Uncaught (in promise) TypeError: Ip is not a function or its return value is not iterable
>     at Sp (index.esm.js:15362:9)
>     at index.esm.js:15328:18
>     at Array.map (<anonymous>)
>     at Op (index.esm.js:15322:22)
>     at index.esm.js:17231:8
>     at index.esm.js:16724:63
>     at Array.map (<anonymous>)
>     at mu._getGroups (index.esm.js:16724:44)
>     at index.esm.js:16710:35
>     at index.esm.js:1626:14

### Steps to reproduce

1. Select an activity
2. Open 'Execution listeners' by clicking 'create'
3. Change 'Listener type' to either 'Expression', 'JavaClass' or 'DelegateExpression'
4. Type something in the inputfield below
5. Delete your input (by pressing backspace several times or select all and then delete)
6. Open console (error doesn't show in Modeler Log)

### Expected behavior

I should be able to delete my inputs without the modeler to break.

### Environment

- OS: [Windows 10]
- Camunda Modeler Version: 5.5.0
- Installed plug-ins: Behaviour was tested without plugins installed

**Comment by marstamm** [comment]
Thank you for reporting this issue. I am able to reproduce it.

## Root Cause

The execution listener is persisted as
```
<camunda:executionListener class="myClass" event="start" />
```
we use the key (`class` in this case) to determine the listener type

When the last character is removed, we do not persist the empty string, resulting in
```
<camunda:executionListener event="start" />
```

Because the type can no longer be determined, the Property panel breaks

## Solution sketch
Persist empty values as empty strings `""` for execution listeners

**Comment by marstamm** [comment]
Same root-cause, different behavior for DMN implementation:

![Recording 2022-11-17 at 17 10 13](https://user-images.githubusercontent.com/21984219/202498149-10856a7c-87b9-47f2-9909-5c9764fbea87.gif)

**Comment by ssternal** [comment]
Thank you guys for the quick response and fix. Do you know when the fix will be incorporated into the Camunda modeler?

**Comment by marstamm** [comment]
Hi @ssternal ,

Thanks for reporting it! we are planning on cutting a patch release for the Camunda Modeler next week.

**Comment by nikku** [comment]
Closed via https://github.com/camunda/camunda-modeler/commit/eeab284c812bf77c7b31b7ae49bcf59364ca18f6.

**Comment by nikku** [comment]
Fixed and released with Camunda Modeler v5.5.1.
